docs/provisioner/08-02-provisioning-gardener.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Provision clusters on GCP, AWS, or AZURE through Gardener
+type: Tutorials
+---
+
+This tutorial shows how to provision clusters with Kyma Runtimes (Runtimes) on Google Cloud Platform (GCP), Microsoft Azure, and Amazon Web Services (AWS) using [Gardener](https://dashboard.garden.canary.k8s.ondemand.com).
+
+## Prerequisites
+
+<div tabs name="Prerequisites" group="Provisioning-Gardener">
+  <details>
+  <summary>
+  GCP
+  </summary>
+  
+  - Existing project on GCP
+  - Existing project on Gardener
+  - Service account for GCP with the following roles:
+      * Service Account Admin
+      * Service Account Token Creator
+      * Service Account User
+      * Compute Admin
+  - Key generated for your service account downloaded in the `json` format
+  </details>
+  
+  <details>
+  <summary>
+  Azure
+  </summary>
+  
+  - Existing project on Gardener
+  - Valid Azure Subscription with the Contributor role and the Subscription ID 
+  - Existing App registration on Azure and its following credentials:
+    * Application ID (Client ID)
+    * Directory ID (Tenant ID)
+    * Client secret (application password)
+
+  </details>
+  
+  <details>
+  <summary>
+  AWS
+  </summary>
+  
+  - Existing project on Gardener
+  - AWS account with added AWS IAM policy for Gardener
+  - Access key created for your AWS user with the following credentials:
+    * Secrete Access Key
+    * Access Key ID
+  
+  > **NOTE:** To get the AWS IAM policy, access your project on Gardener, navigate to the **Secrets** tab, click on the help icon on the AWS card, and copy the `json` policy. 
```

Comment:
I'm not correcting `json` formatting here as I am not sure if that is how it is written on Gardener UI or sth so I leave it to you.